_discsimmodule.c

```diff
@@ -338,8 +350,9 @@ Simulator_init(Simulator *self, PyObject *args, PyObject *kwds)
     int sim_ret;
     static char *kwlist[] = {"sample", "event_classes", "num_loci", 
             "num_parents", "max_population_size", "max_occupancy", 
-            "dimension", "simulate_pedigree", "random_seed", "torus_diameter",
-            "pixel_size", "recombination_probability", NULL}; 
+            "dimension", "simulate_pedigree", "simulate_kingman", 
+            "random_seed", "torus_diameter", "pixel_size", 
+            "recombination_probability", "r", "Ne", NULL}; 
```

Comment:
You're right, I'll change them. I was following the coalescent literature notation, and not being consistent with your code.

The arg_recombination_rate is in principle a free parameter, which is why I think it should be separate. I've set it at  1 / (L Ne) to more or less match the recombination probability of 0.5 / L in the spatial phase, because the rescaled recombination rate is Ne * r.